### DISCUSSION TITLE
HiGHS not running in parallel on Ubuntu 18.04.2 LTS with 16-core Intel Xeon CPU

### DISCUSSION THREAD

**Discussion by n-tateiwa** [ERGO-Code/HiGHS#1236] [Q&A] [answered]
I am running HiGHS version 1.5.0 on an Ubuntu 18.04.2 LTS machine with a 16-core Intel Xeon Gold 6130 CPU. However, when I even try to run HiGHS with the --parallel on flag, it does not run in parallel. The full command I tried is as follows.

```sh
highs bnatt500.mps --parallel on
```

Is there a reason why HiGHS is not running in parallel on my machine? Is there anything I can do to fix this problem?

Any help or advice on this matter would be greatly appreciated.

**Comment by jajhall** [comment]
What class of problem is `bnatt500.mps`?

Comments on parallelism are given in

https://ergo-code.github.io/HiGHS/dev/parallel/

**Comment by n-tateiwa** [comment]
Thank you for your reply!

bnatt500 is ILP with only binary variables.
https://miplib.zib.de/instance_details_bnatt500.html

**Comment by jajhall** [comment, marked as answer]
So, as per my observations in 
https://ergo-code.github.io/HiGHS/dev/parallel/
the MIP solver makes little use of parallelism - yet. We still beat SCIP, though!

**Comment by n-tateiwa** [comment]
Thank you for sharing the information!
